bears/general/SpaceConsistencyBear.py

```diff
@@ -38,7 +41,52 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
-        for line_number, line in enumerate(file, start=1):
+        def get_blanklines_nr_start():
+            line_nr_start = False
+            if not allow_trailing_blanklines:
+                rev_enumerated_zip_obj = zip(reversed(range(1, len(file) + 1)),
+                                             reversed(file))
+                rev_enumerated_tuple = tuple(rev_enumerated_zip_obj)
```

Comment:
kind of defeats the purpose of having a `zip/generator` like object which is highly efficient than a `list like` object